ci-operator/step-registry/ipi/conf/gcp/user-labels-tags/ipi-conf-gcp-user-labels-tags-commands.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/user-labels-tags.yaml.patch"
+cat > "${PATCH}" << EOF
+platform:
+  gcp:
+EOF
+
+# user labels
+i=0
+printf '%s' "${USER_LABELS:-}" | while read -r KEY VALUE || [ -n "${KEY}" ]
```

Comment:
What is `|| [ -n "${KEY}" ]` used for?

Reply:
I tried locally, and if without it, the last key & value won't be dealt with.